src/Sylius/Bundle/PaymentBundle/DependencyInjection/Configuration.php

```diff
@@ -57,6 +57,14 @@ public function getConfigTreeBuilder(): TreeBuilder
                         ->end()
                     ->end()
                 ->end()
+                ->arrayNode('payment_request')
+                    ->addDefaultsIfNotSet()
+                    ->children()
+                        ->arrayNode('states_that_should_be_cancelled_when_payment_request_has_changed')
```

Comment:
Maybe simple:
```suggestion
                        ->arrayNode('states_to_be_cancelled')
```
or at least:
```suggestion
                        ->arrayNode('states_to_be_cancelled_when_payment_method_changed')
```